tasks/main.yml

```diff
@@ -18,11 +18,13 @@
 
 - name: open permissions on settings file.
   file:
-    path: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}/settings.php"
+    path: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}/"
```

Comment:
I'm pretty sure the `file` module won't like this, but figured this would get my idea across.